backend/src/journals/journals.service.ts

```diff
@@ -355,8 +364,9 @@ export class JournalsService {
             JournalInfoForList.getKeysForJournalTable(),
         );
         const findResult = await this.journalsDogsService.find({ where: { dogId }, select: ['journalId'] });
-        //TODO: jd -> journal Data 변수명 수정, 로직 개선
-        const journalCntForFirstRow = findResult.findIndex((jd) => jd.journalId === journalInfos[0].journalId);
+        const journalCntForFirstRow = findResult.findIndex(
+            (journalData) => journalData.journalId === journalInfos[0].journalId,
+        );
```

Comment:
```suggestion
        const journalCntForFirstRow = findResult.findIndex(
            ({ journalId }) => journalId === journalInfos[0].journalId,
        );
```
구조 분해 할당을 하면 더 간편하게 나타낼 수 있어요

Reply:
오 감사합니다!!
getJournalList 함수는 두개로 나눠진 쿼리를 하나로 합치면서 코드가 아예 바뀔것같아서 
지금 바꿔도 다음 커밋에서 날아갈거라 일단 그대로 두곘습니다..!
분해 할당 계속 까먹는데 알려주셔서 감사해요 ㅎ.ㅎ